### ISSUE TITLE
Missing script tag for external js file

### ISSUE THREAD

**Issue by yangc95** [Acadecraft123/How-to-Build-a-Loading-Screen-for-Your-Web-App#3]
In the article, the css file is named styles.css. (We should also comment the file name before the css like how you've done for the other js and html code blocks)

To include external js files we should add this line within the head tags:
`<script type="text/javascript" src="scripts.js"></script>`

**Comment by NaincyKumariKnoldus** [comment]
Hey @yangc95 I liked to do this. I have done this and raised a PR now. Please Review this.
Thanks!

**Comment by NaincyKumariKnoldus** [comment]
Hey @yangc95 made the requested changes. Have a look please.

**Comment by NaincyKumariKnoldus** [comment]
Hey @yangc95 again made the changes. Review that please.

**Comment by NaincyKumariKnoldus** [comment]
Hey @yangc95 You have approved this thanks! But it is not merged yet.

**Comment by yangc95** [comment]
> Hey @yangc95 You have approved this thanks! But it is not merged yet.

@NaincyKumariKnoldus Yup! Someone with permission is able to merge this. Unfortunately, I do not have those permissions. Would you happen to know who in Acadecraft does?
